extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/DevConsoleCORSFilter.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.vertx.http.runtime.devmode;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.jboss.logging.Logger;
+
+import io.quarkus.vertx.http.runtime.cors.CORSConfig;
+import io.quarkus.vertx.http.runtime.cors.CORSFilter;
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.ext.web.RoutingContext;
+
+public class DevConsoleCORSFilter implements Handler<RoutingContext> {
+    private static final Logger LOG = Logger.getLogger(DevConsoleCORSFilter.class);
+
+    private static final String HTTP_PORT_CONFIG_PROP = "quarkus.http.port";
```

Comment:
I think `quarkus.http.ssl-port` (8443) should be fetched and used.

`HTTPS_LOCAL_HOST + ":" + httpPort` on line 31 doesn't look right, https doesn't run on 8080.

https://localhost:8443/q/dev is the default address for Dev UI on https.

Reply:
Tried to update stuff using https://localhost:8443/q/dev/io.quarkus.quarkus-vertx-http/config and the console reports `403 Forbidden`. So config update using DevUI on https is blocked now.

Reply:
@rsvoboda Good catch, fixed